client/src/components/pages/AboutUs/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Navbar, Footer } from '../../utils';
+
+import styles from './about.module.css';
+
+const About = () => {
+  return (
+    <div>
+      <Navbar transparent={false} />
```

Comment:
the default value for the navbar is false, so there is no need passing it here 
```suggestion
      <Navbar />
```